pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -50,7 +50,7 @@ func TestApplicationSelectorByName(t *testing.T) {
 	backingServiceResourceRef := "backingServiceRef"
 	applicationResourceRef := "applicationRef"
 	f := mocks.NewFake(t, reconcilerNs)
-	f.AddMockedUnstructuredServiceBindingRequest(reconcilerName, backingServiceResourceRef, applicationResourceRef, deploymentsGVR, nil)
+	f.AddMockedUnstructuredServiceBinding(reconcilerName, backingServiceResourceRef, applicationResourceRef, deploymentsGVR, nil)
```

Comment:
We might want to log an issue for changes in mocks names. Not necessarily to be addressed in this PR

Reply:
Some of the variable names could remain, they might make things more expressive?